configs/filter.yaml

```diff
@@ -1,3 +1,7 @@
+#tenants:
+#    tenant_id:
+#        - <tenant_id1>
+#        - <tenant_id2>
```

Comment:
Will be done after https://github.com/MirantisWorkloadMobility/CloudFerry/pull/303